pkg/search/postgres/common.go

```diff
@@ -546,11 +547,17 @@ func standardizeQueryAndPopulatePath(q *v1.Query, schema *walker.Schema, queryTy
 	return parsedQuery, nil
 }
 
-func standardizeSelectQueryAndPopulatePath(q *v1.Query, schema *walker.Schema, queryType QueryType) (*query, error) {
+func standardizeSelectQueryAndPopulatePath(ctx context.Context, q *v1.Query, schema *walker.Schema, queryType QueryType) (*query, error) {
 	nowForQuery := time.Now()
+
+	var err error
+	q, err = scopeContextToQuery(ctx, q)
+	if err != nil {
+		return nil, err
+	}
+
 	standardizeFieldNamesInQuery(q)
 	innerJoins, dbFields := getJoinsAndFields(schema, q)
```

Comment:
Should we check if innerJoins and dbFields are nil/empty here? I think this could happen if selects and matchFields are unrelated to the schema being queried.

Reply:
We do not need to. The individual parsers below already tackle it.